src/reporter.ts

```diff
@@ -35,17 +36,38 @@ export class Reporter {
   private suiteIds: string[];
   private testItems: TestItem[];
   private testData: { [name: string]: string };
+  private testCaseIds: Map<string, string>;
 
   constructor(config: ReportPortalConfig) {
     this.noColors = false;
     this.suiteIds = [];
     this.testItems = [];
     this.testData = {};
+    this.testCaseIds = new Map<string, string>();
 
     const agentInfo = getAgentInfo();
 
     this.config = config;
     this.client = new RPClient(config, agentInfo);
+    this.registerRPListeners();
```

Comment:
Does this work as expected?
It looks like testCafe is copying our reporter object to its own reporter instance and it will never be available to the testCafe runner.
So in this case you need to register additional listeners inside `reportTaskStart` (see the [link](https://github.com/reportportal/agent-js-testcafe/pull/10/files#diff-f5d7562fe14b5c4c08e44380f423efbc7bdf7ccdd0e799917503eec5334ed633R61)).
@chivekrodis can explain this in more detail.